Great. Don't think that 

```bash
Caused by: org.eclipse.jgit.errors.RevWalkException: Walk failure.
	at org.eclipse.jgit.revwalk.RevWalk.iterator(RevWalk.java:1259)
	at pl.project13.jgit.DescribeCommand.findCommitsUntilSomeTag(DescribeCommand.java:404)
	... 32 more
Caused by: org.eclipse.jgit.errors.MissingObjectException: Missing commit 721db2b0560630faa0ae9e7693c800fc9e8dcd54
	at org.eclipse.jgit.internal.storage.file.WindowCursor.open(WindowCursor.java:149)
	at org.eclipse.jgit.revwalk.RevWalk.getCachedBytes(RevWalk.java:883)
	at org.eclipse.jgit.revwalk.RevCommit.parseHeaders(RevCommit.java:145)
	at org.eclipse.jgit.revwalk.PendingGenerator.next(PendingGenerator.java:[148](https://github.com/sweble/sweble-wikitext/actions/runs/8642953682/job/23695614231#step:5:149))
	at org.eclipse.jgit.revwalk.StartGenerator.next(StartGenerator.java:183)
	at org.eclipse.jgit.revwalk.RevWalk.next(RevWalk.java:421)
	at org.eclipse.jgit.revwalk.RevWalk.iterator(RevWalk.java:1257)
	... 33 more
```

is actually related.

--- a/.github/workflows/maven.yml
+++ b/.github/workflows/maven.yml
@@ -17,6 +17,9 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+        with:
+          path: tooling/sweble-wikitext
+          fetch-depth: 0
       - uses: actions/cache@v3
         with:
           path: ~/.m2/repository

--- a/.github/workflows/maven.yml
+++ b/.github/workflows/maven.yml
@@ -18,7 +18,6 @@ jobs:
     steps:
       - uses: actions/checkout@v3
         with:
-          path: tooling/sweble-wikitext
           fetch-depth: 0
       - uses: actions/cache@v3
         with:

